deploy/helm/templates/logs-daemon-set.yaml

```diff
@@ -60,8 +60,9 @@ spec:
                 values:
                 - fargate
       {{- end }}
-      {{- if .Values.otel.logs.nodeSelector }}
       nodeSelector:
+        kubernetes.io/os: linux
+      {{- if .Values.otel.logs.nodeSelector }}
```

Comment:
Why this change? Also, isn't it already covered by the `nodeAffinity`?

Reply:
Well windows node require nodeSelector and affinity wasn't enough for him, So I tried to be consistent.

Reply:
Well, we played with this error https://karpenter.sh/v0.29/troubleshooting/#windows-pods-are-failing-with-failedcreatedpodsandbox and we tried to follow instructions with @jaroslav-fedor-swi  and the only solution was for us move from affinity to node selector. It's hard to say whether it's really needed, but without rights I cannot change AIM roles.

Reply:
Jaro found information where it's explicitly mentioned nodeSelector and without nodeSelector is really did not work. 

https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#remove-windows-support-data-plane